openid-4-verifiable-presentations-1_0.md

```diff
@@ -528,11 +528,13 @@ If a `:` character is not present in the Client Identifier, the Wallet MUST trea
 
 For example, if an Authorization Request contains `client_id=example-client`, the Wallet would interpret the Client Identifier as referring to a pre-registered client.
 
+If the Client Identifier begins with `https://` the full Client Identifier URL represents an Entity Identifier defined in OpenID Federation [@!OpenID.Federation] and MUST be processed as though it was prefixed with `openid_federation` as described below.
```

Comment:
> I personally don't have an issue with OpenID4VP defining the default behavior of https URLs as long as it's defined only within the context of OpenID4VP and doesn't spill out to unrelated communities and ecosystems.

if this is the case, it must be much more explicit that this https:// behavior is only for openid4vp context